src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
```

Comment:
Consider using `const` for the `sum` variable since it's not being reassigned, but rather being used to store the final calculation before returning. This adheres to checklist item 3.